css/media.css

```diff
@@ -0,0 +1,388 @@
+@media (max-width: 1220px) {
+   .welcome-page {
+     width: 100%;
+     margin: 0 auto;
+   }
+ }
+
+ @media (max-width: 1036px) {
+   .header__menu_link {
+     font-size: 12px;
+     line-height: 15px;
+   }
+
+   .link__item {
+     font-size: 12px;
+     line-height: 15px;
+   }
+ }
+
+ @media (max-width: 856px) {
+   .header__menu_link {
+     display: none;
+   }
+
+   .welcome-page__speed-car {
+     width: 80%;
+   }
+
+   .welcome-page__speed-sec {
+     font-size: 28px;
+     line-height: 33px;
+   }
+
+   .link-none {
+     display: none;
+   }
+
+   .welcome-page__block {
+     width: 50%;
+     margin: 0;
+   }
+ }
+
+ @media (max-width: 768px) {
+   .header {
+     box-sizing: border-box;
+     padding: 24px 33px 0 15px;
+   }
+
+   .header__menu {
+     display: none;
+   }
+
+   .link-none {
+     display: none;
+   }
+
+   .link__img {
+     width: 35.23px;
+     height: 22.02px;
+     margin-left: auto;
+     margin-right: 0;
+   }
+
+   .car__title {
+     margin-top: 80px;
+   }
+
+   .car__subtitle {
+     position: relative;
+     bottom: 10px;
+     padding-left: 16px;
+   }
+
+   .car {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)